translations/ja.json

```
@@ -688,10 +688,10 @@
 	"Things to try without Kubernetes ...": "Kubernetes なしで試すべきこと ...",
 	"This addon does not have an endpoint defined for the 'addons open' command.\nYou can add one by annotating a service with the label {{.labelName}}:{{.addonName}}": "このアドオンは 'addons open' コマンド用に定義されたエンドポイントがありません。\nサービスに {{.labelName}}:{{.addonName}} ラベルを付与することでエンドポイントを追加できます",
 	"This can also be done automatically by setting the env var CHANGE_MINIKUBE_NONE_USER=true": "これは環境変数 CHANGE_MINIKUBE_NONE_USER=true を設定して自動的に行うこともできます",
-	"This cluster was created before minikube v1.26.0 and doesn't have cri-docker installed. Please run 'minikube delete' and then start minikube again": "",
+	"This cluster was created before minikube v1.26.0 and doesn't have cri-docker installed. Please run 'minikube delete' and then start minikube again": "このクラスターは minikube v1.26.0 より前に作成され、cri-docker がインストールされていません。`minikube delete` を実行してから、minikube を再起動してください。",
```

```suggestion
	"This cluster was created before minikube v1.26.0 and doesn't have cri-docker installed. Please run 'minikube delete' and then start minikube again": "このクラスターは minikube v1.26.0 より前に作成され、cri-docker がインストールされていません。`minikube delete` を実行してから、再度 minikube を起動してください。",
```
`再起動` is confusing about which should we use `start` or `restart` ( I knew minkube don't have `restart` ), so it's good to use `再度 minikube を起動する` for clarify.

---

good point.